src/microbe_stage/MicrobeColony.cs

```diff
@@ -172,4 +187,17 @@ private void UpdateHexCount()
             hexCount += member.EngulfSize;
         }
     }
+
+    private void UpdateCanEngulf()
+    {
+        canEngulf = false;
+
+        foreach (var member in ColonyMembers)
+        {
+            canEngulf = member.CanEngulf;
```

Comment:
The compiler might optimize this, but I think
```c#
if (!member.CanEngulf)
    continue;
canEngulf = true;
break;
```
would be preferable to show that the canEngulf variable only really needs to be written to once.